modules/batch/src/main/java/com/devonfw/module/batch/common/base/SpringBootBatchCommandLine.java

```diff
@@ -172,7 +173,7 @@ public void execute(Operation operation, List<String> configurations, String job
     SpringApplication app = new SpringApplication(configurationClasses);
 
     // no (web) server needed
-    app.setWebEnvironment(false);
+    app.setWebApplicationType(WebApplicationType.NONE);
```

Comment:
Fine fix. However, this reminds me of the fact that our batch still needs rework and polising as this is old fashion style code. But removing the spring XMLs and moving to a modern spring-boot only config is not in the scope of this PR, so all perfectly right.